.changeset/violet-trees-help.md

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
if we drop support for Node 18, we need to make this a major release.

Also I noticed we made changes in more packages than listed in the changeset.

Reply:
Removed other changes for now so we should be good.